Common/src/linear_algebra/CSysSolve.cpp

```diff
@@ -452,7 +452,9 @@ unsigned long CSysSolve<ScalarType>::FGMRES_LinSolver(const CSysVector<ScalarTyp
 
     /*---  Modified Gram-Schmidt orthogonalization ---*/
 
+    SU2_OMP_PARALLEL_(if (!omp_in_parallel()))
```

Comment:
How do I interpret this?
omp_in_parallel returns nr of active levels, so ModGramSchmidt is not allowed to use nesting?

Reply:
You can interpret as "make it parallel if not parallel yet", but it's broken because of shared access to the arguments of the function.
I need to think a bit more on this.